src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -53,13 +60,19 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
      */
     onYearChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
         const nextYear = parseInt(event.target.value);
-        const election = this.props.electionType.elections.find((election) => election.year === nextYear);
-        const votes = this.props.votes.filter((vote) => vote.electionYear === nextYear);
-        const metrics = this.props.metrics.filter((metric) => metric.electionYear === nextYear);
+        let election = this.props.electionType.elections.find((election) => election.year === nextYear);
+        let votes = this.props.votes.filter((vote) => vote.electionYear === nextYear);
+        let metrics = this.props.metrics.filter((metric) => metric.electionYear === nextYear);
```

Comment:
I would have preferred having a code path for this instead of reusing the variables -- the memory expenditure isn't really a problem, especially if you check for "should I merge" before the code is executed.